java/impl/gcp/src/main/java/co/worklytics/psoxy/SecretManagerConfigService.java

```diff
@@ -82,7 +97,7 @@ private String parameterName(ConfigProperty property) {
         if (StringUtils.isBlank(this.namespace)) {
             return property.name();
         } else {
-            return String.join("_", this.namespace, property.name());
+            return this.namespace + property.name();
```

Comment:
I'd prefer that the namespace doesn't contains `_` as it is used as separator

Reply:
that's the default for now for backwards compatibility; this gives flexibility, as you can override with whatever you want. If you pass explicit value via ENV VARS, that value is used directly as `namespace`; if you don't pass anything, by default will now be `PSOXY_` (instead of `PSOXY` and `_` getting appended). This allows people to use a path-like structure `PSOXY/` if they want.